src/OpenTelemetry.Exporter.Geneva/Metrics/TlvMetricExporter.cs

```diff
@@ -407,9 +407,15 @@ private static void SerializeMonitoringAccount(string monitoringAccount, byte[]
 
 #if EXPOSE_EXPERIMENTAL_FEATURES
     [MethodImpl(MethodImplOptions.AggressiveInlining)]
-    private static void SerializeExemplars(Exemplar[] exemplars, byte[] buffer, ref int bufferIndex)
+    private static void SerializeExemplars(ReadOnlyExemplarCollection exemplars, byte[] buffer, ref int bufferIndex)
```

Comment:
nit: Since `ReadOnlyExemplarCollection` is a `readonly struct` and its size is greater than `IntPtr.Size`, use the `in` modifier for better performance. Check this [doc](https://devblogs.microsoft.com/premier-developer/the-in-modifier-and-the-readonly-structs-in-c/) for more details.

```suggestion
    private static void SerializeExemplars(in ReadOnlyExemplarCollection exemplars, byte[] buffer, ref int bufferIndex)
```